examples/skymapStitch.py

```diff
@@ -187,11 +186,25 @@ def getSkyMapCoaddUtil(ctrCoord, width, height, filt, source, mapType, patchType
     source = "/lsst7/releaseW13EP"
     mapType = "deepCoadd_skyMap"
     patchType = "deepCoadd"
+
+    #raA = afwGeom.Angle(0.0, afwGeom.degrees)
+    #decA = afwGeom.Angle(0.01667, afwGeom.degrees)
+    #ctrCoord = afwCoord.Coord(raA, decA, 2000.0)
     ctrCoord = afwCoord.Coord("1:01:00", "00:01:00", 2000.0)
-    expo = getSkyMap(ctrCoord, 4000, 300, 'r', source, mapType, patchType)
+    f = 'r'
+    w = 4000
+    h = 300
+    expo = getSkyMap(ctrCoord, w, h, f, 'pixel', source, mapType, patchType)
     expo.writeFits("getSkyMap.fits")
-    expo = getSkyMapGoodPixelCopy(ctrCoord, 4000, 300, 'r', source, mapType, patchType)
+    expo = getSkyMapGoodPixelCopy(ctrCoord, w, h, f, source, mapType, patchType)
     expo.writeFits("getSkyMapGoodPixelCopy.fits")
-    expo = getSkyMapCoaddUtil(ctrCoord, 4000, 300, 'r', source, mapType, patchType)
+    expo = getSkyMapCoaddUtil(ctrCoord, w, h, f, source, mapType, patchType)
     expo.writeFits("getSkyMapCoaddUtil.fits")
 
+    decA = afwGeom.Angle(0.1, afwGeom.degrees)
+    for rai in range(1, 3595):
+        raf = float(rai) / 10.0
```

Comment:
I'd rather we imported `division` from `__future__` at the top so we didn't need the extra cruft to force a floating point division.